development/otp-input-app/pages/otp-input/index.mdx

```diff
@@ -0,0 +1,99 @@
+import ReuseNumberExample from "../../components/otp-input/ReuseNumberExample/ReuseNumberExample.tsx";
+import ReuseNumberWithPasteExample from "../../components/otp-input/ReuseNumberWithPasteExample/ReuseNumberWithPasteExample.tsx";
+
+
+## Table of contents
+
+## List of exported Components:
+
+### HeadlessOtpInput 
+
+This is the OTP input component with no inherit styles. user will have to provide styles to the component.
+
+This OTP input component we can customize the number of inputs based on requirements we can pass the prop `numInputs` with value.
+
+We need to use the HTML input element to render the boxes. and if we need to add wrapper to input components with help of renderInput prop. and `InputProps` It allows the basic HTML input element allowed props.
+
+Here is the list of all available props for the HeadlessOtpInput
+
+`numInputs:number`: Number of OTP inputs to be rendered.
+
+`renderInput:function`: A function that returns the input that is supposed to be rendered for each of the input fields. The function will get two arguments: `inputProps` and index. `inputProps` is an object that contains all the props that should be passed to the input being rendered index is the index of the input being rendered.
+
+`onChange:function`: Returns OTP code typed in inputs.
+
+`value: string/number`: The value of the OTP passed into the component.
+
+`placeholder:string`: Specify an expected value of each input. The length of this string should be equal to numInputs.
+
+`renderSeparator:component/function`: Provide a custom separator between inputs by passing a component. For instance, <span>-</span> would add - between each input.
+
+`containerStyle:style (object)/className (string)`: Style applied or class passed to container of inputs.
+
+`inputStyle:style (object)/className (string)`: Style applied or class passed to each input.
+
+`inputType:<input> type`: The type of the input that will be passed to the input element being rendered.
+
+`shouldAutoFocus: boolean`: This is optional prop and it Auto focuses input on initial page load.
+
+
+
+#### Sample Code Demo
+
+<Sample/>
+
+```js path=../../components/otp-input/Sample name=Sample
+```
+
+
+### ReuseOtpInput
+
+The **`ReuseOtpInput`** component extends the functionality of **`HeadlessOtpInput`** by providing additional features and default styling options, which can be overridden to customize components according to your specific use case.
+
+User can pass the custom styles using this prop `wrapperClassName`  and `inputClassName`. 
+
+By leveraging these props, users can fully customize the **ReuseOtpInput** component to suit their specific requirements and design preferences.
+
+
+#### ReuseExample
+
+<ReuseExample/>
+
+```js path=../../components/otp-input/ReuseExample name=ReuseExample
+```
+
+#### ReuseNumberExample
+
+This example demonstrating the accepting the only number type values. and we have added css to hide the default html number arrows (up and down).
```

Comment:
This example demonstrates an OTP component that accepts only number type values. 
` We have added css to hide the default html number arrows (up and down)`

Change to above